### PR TITLE
PyTorch Writing Algorithm

### PR DESCRIPTION
**Pull request by LouisSzeto**
#### Description
Add tutorial for `PyTorch` library on [Writing Algorithm/Machine Learning](https://github.com/QuantConnect/Documentation/tree/master/02%20Writing%20Algorithms/30%20Machine%20Learning/03%20Popular%20Libraries).

#### Motivation and Context
Easy implementation example for users.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [X] New feature (non-breaking change which adds functionality)
- [X] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [X] My code follows the code style of this project.
- [X] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [X] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: 02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/06 PyTorch/02 Import Libraries.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<p>Import the <code>torch</code>, and <code>joblib</code> libraries.</p>
```
Comment (DerekMelchin) [inline 02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/06 PyTorch/02 Import Libraries.html:1]: Remove `,`

File: 02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/06 PyTorch/05 Train Models.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+history = self.History[TradeBar](self.symbol, training_length, Resolution.Daily)
+for trade_bar in history:
+    self.training_data.Add(trade_bar.Close)</pre>
+</div>
+
+<h4>Define a Training Method</h4>
+<p>To train the model, define a method that fits the model with the training data.</p>
+<div class="section-example-container">
+    <pre class="python">def get_features_and_labels(self, n_steps=5):
+    training_df = list(self.training_data)[::-1]
```
Comment (DerekMelchin) [inline 02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/06 PyTorch/05 Train Models.html:17]: `training_df` <- I don't think this is a DataFrame, let's change the variable name

File: 02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/06 PyTorch/06 Predict Labels.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<p>To predict the labels of new data, in the OnData method, get the most recent set of features and then call the predict method.</p>
```
Comment (DerekMelchin) [inline 02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/06 PyTorch/06 Predict Labels.html:1]: OnData => `OnData`

Comment (DerekMelchin) [reply 02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/06 PyTorch/06 Predict Labels.html:1]: It says "call the predict method", but there is no `predict` method in the following snippet

**Review by DerekMelchin — APPROVED** [review]
